app/models/post.rb

```diff
@@ -0,0 +1,4 @@
+class Post < ActiveRecord::Base
+  has_attached_file :image, :styles => { :large => "600x600>", :medium => "400x400>" }, :default_url => "/images/:style/missing.png"
```

Comment:
Metrics/LineLength: Line is too long. [132/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)